### PR TITLE
Refactor/misc better improvements

### PR DESCRIPTION
**Pull request by macintoshpie**
#### What's this PR do?
- add energy, cost, and ghg savings specific to electricity and fuel (nat gas)
  - if BETTER reports that it failed to create a model for the energy resource (electricity or nat gas) then set those energy-specific savings to NULL. Also, create a warning message for the property (see screenshot)
- remove street address from the BuildingSync file sent to BETTER (i.e. no longer require that SEED properties have address line 1)

#### How should this be manually tested?
- Run a better analysis
- update your column profile to include the new BETTER columns mentioned above (e.g. BETTER Potential Electricity Energy Savings)
- verify that if the analysis failed to create a model for natural gas or electricity (look at the HTML page to see) then the energy-specific savings in the inventory list should be empty.

#### What are the relevant tickets?
#2884 
#2886 
#2840 

#### Screenshots (if appropriate)
Screen Shot 2021-09-20 at 2 15 41 PM
Screen Shot 2021-09-27 at 11 28 02 AM

### CODE REVIEW

File: seed/analysis_pipelines/better/pipeline.py [hunk 1]
```diff
@@ -311,21 +311,48 @@ def _process_results(self, analysis_id):
     ]
 
     # gather all columns to store
+    BETTER_VALID_MODEL_E_COL = 'better_valid_model_electricity'
+    BETTER_VALID_MODEL_F_COL = 'better_valid_model_fuel'
     column_data_paths = [
         ExtraDataColumnPath(
-            'better_cost_savings_combined',
-            'BETTER Potential Cost Savings (USD)',
-            'assessment.assessment_energy_use.cost_savings_combined'
```
Comment (macintoshpie) [inline seed/analysis_pipelines/better/pipeline.py:318]: @nllong these changes removes the combined cost, ghg, energy usage columns from the inventory list -- is that ok? we're now pulling out electricity/nat-gas specific values.

**Comment by RDmitchell** [comment]
@macintoshpie -- what values are removed and what are added to the Inventory list?

**Comment by macintoshpie** [comment]
@RDmitchell that's fair, I did remove some of the values which you see highlighted there in your screenshot. I will add the back!

**Comment by RDmitchell** [comment]
@macintoshpie -- thanks ... just seems like it would be good to be able to match the values in the inventory list with the values on the report.

**Review by nllong — APPROVED** [review]
This looks great. Imported our test data and ran it through BETTER. 

image

I noticed the header of the BSync files still show BSync 2.2, is that correct?

```
<auc:BuildingSync xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:auc="http://buildingsync.net/schemas/bedes-auc/2019" xsi:schemaLocation="http://buildingsync.net/schemas/bedes-auc/2019 https://raw.github.com/BuildingSync/schema/v2.2.0/BuildingSync.xsd" version="2.2.0">
```

**Comment by macintoshpie** [comment]
> I noticed the header of the BSync files still show BSync 2.2, is that correct?

Yeah that's what we generate, we even have a test which validates the bsync file we generate against the 2.2 schema. We should be fine bumping to 2.3 as well. Is that something we'd want to do?

**Comment by nllong** [comment]
> > I noticed the header of the BSync files still show BSync 2.2, is that correct?
> 
> Yeah that's what we generate, we even have a test which validates the bsync file we generate against the 2.2 schema. We should be fine bumping to 2.3 as well. Is that something we'd want to do?

It would be great to bump to 2.3 if we have time today... whatcha think?

**Pull request merged**